coconut/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: aXe future tools
```

Comment:
We should capitalize `Future Tools` to be consistent with other pages, otherwise looks good!